pip/src/fs.rs

```diff
@@ -35,20 +35,20 @@ struct FsHooks {
 
 #[derive(Debug)]
 struct Entry {
-    entry_type: EntryType,
+    ty: EntryType,
```

Comment:
These name changes are just me responding to clippy 1.75. There are other new clippy warnings-as-errors in the repo but we'll get to those in a separate PR.